content/expressions/methods.md

```diff
@@ -10,7 +10,7 @@ toc: true
 
 All Pony code that actually does something, rather than defining types etc, appears in named blocks which are referred to as methods. There are three kinds of methods: functions, constructors, and behaviours. All methods are attached to type definitions (e.g. classes) - there are no global functions.
 
-Behaviours are used for handling asynchronous messages sent to actors. We'll look at those later.
+Behaviours are used for handling asynchronous messages sent to actors, which we've seen in the "Types" chapter [when we talked about actors](../types/actors.md).
```

Comment:
that wont work for linking. you need to use the url or you can use the hugo rel/absolute link commands.